main.js

```diff
@@ -104,6 +106,39 @@ function initApp() {
       app.quit();
     });
   });
+
+  // Delay, then start qtum-qt
+  server.emitter.on(ipcEvent.QTUMD_KILLED, () => {
+    setTimeout(() => {
+      require('./src/start_wallet');
+    }, 4000);
+  });
+}
+
+function showLaunchQtumWalletDialog() {
+  app.focus();
+  dialog.showMessageBox({
+    type: 'question',
+    buttons: ['Cancel', 'Launch'],
+    title: 'Launch Qtum Wallet?',
+    message: 'You are about to launch the Qtum Wallet. Bodhi will need to be shutdown to run Qtum Wallet. To run Bodhi again, you will have to close the Qtum Wallet and run Bodhi again.\n\nBodhi will shutdown shortly. Are you sure to want to continue?',
```

Comment:
localization seems necessary on electron side now

Reply:
true. we have quite a lot of strings for this now. let me look at possible solutions.